linera-service/src/linera.rs

```diff
@@ -1691,40 +1695,47 @@ async fn main() -> Result<(), anyhow::Error> {
         }
 
         ClientCommand::Net(net_command) => match net_command {
-            NetCommand::Up => {
+            NetCommand::Up { wallets } => {
                 let network = Network::Grpc;
                 let mut net = LocalNetwork::new(Database::RocksDb, network, 1)?;
                 let client1 = net.make_client(network);
-                let client2 = net.make_client(network);
 
                 net.generate_initial_validator_config().await?;
                 client1.create_genesis_config().await?;
-                client2.wallet_init(&[]).await?;
 
                 // Create initial server and client config.
                 net.run().await?;
-                let net_path = net.net_path();
+
+                for wallet in 0..*wallets {
+                    let extra_wallet = net.make_client(network);
+                    extra_wallet.wallet_init(&[]).await?;
+                    eprintln!("\nExtra wallet {}:", wallet + 1);
+                    eprintln!(
+                        "export LINERA_WALLET=\"{}\"",
```

Comment:
The display is going to look weird when there are several wallets.

Reply:
It looks good IMO:

```
Extra wallet 1:
export LINERA_WALLET="/tmp/.tmpmVS0DP/wallet_1.json"
export LINERA_STORAGE="rocksdb:/tmp/.tmpmVS0DP/client_1.db"

2023-09-23T17:51:05.379369Z  INFO linera: Saved user chain states

Extra wallet 2:
export LINERA_WALLET="/tmp/.tmpmVS0DP/wallet_2.json"
export LINERA_STORAGE="rocksdb:/tmp/.tmpmVS0DP/client_2.db"

2023-09-23T17:51:05.432417Z  INFO linera: Saved user chain states

Extra wallet 3:
export LINERA_WALLET="/tmp/.tmpmVS0DP/wallet_3.json"
export LINERA_STORAGE="rocksdb:/tmp/.tmpmVS0DP/client_3.db"


Linera net directory available at: /tmp/.tmpmVS0DP
To configure your Linera client for this network, run:

export LINERA_WALLET="/tmp/.tmpmVS0DP/wallet_0.json"
export LINERA_STORAGE="rocksdb:/tmp/.tmpmVS0DP/client_0.db"
```